### ISSUE TITLE
[suggestion] mining should wait for item pick up + auto eat

### ISSUE THREAD

**Issue by MrDodojo** [cabaletta/baritone#508]
The baritone miner should wait before the items are picked up.

another suggestion,
auto eating.

**Comment by TypicalHog** [comment]
Yes, I've seen him mine the block walk 10 blocks away to mine the other ores and then stop and come back to pick up the other ore. Sometimes we went back and forth 2-3 times.

**Comment by Martysh12** [comment]
You should DEFINITELY add that, baritone was mining, for diamonds, of course, it mined 1 block str8 down, and it missed the diamond.

**Comment by StijnSimons** [comment]
This is due to server lag; baritone broke the block and doesn’t see a drop appearing so it continues its path.

**Comment by Martysh12** [comment]
Ohhh, OK thank you 👌

вс, 4 авг. 2019 г., 22:33 Baddeveloper <notifications@github.com>:

**Comment by AlexApps99** [comment]
@MrDodojo please close this issue as it's due to server-side lag and cannot be patched.
